### ISSUE TITLE
Grafana Alert-Email notifications not working on a non-tls smtp server

### ISSUE THREAD

**Issue by Jhonrubia-LAS** [grafana/grafana#69654]
### 
# What went wrong?

**What happened**:
We have configured the SMTP connection in our Grafana installation against an internal server with the following configuration:
Once this is done, we have created a point of contact using email as a mechanism.
When launching the notification test, the following error is displayed:
![image](https://github.com/grafana/grafana/assets/65592618/cea69e0c-8b92-45d1-a4cc-21a5a1a5174c)
The following messages are displayed in the grafana log:
```
logger=ngalert.notifier.email t=2023-06-06T15:08:14.761627379Z level=error msg="Missing receiver"
logger=ngalert.notifier.email t=2023-06-06T15:08:14.761785522Z level=error msg="Missing group labels"
```
And no mail is received on the destination account.

  - 

**What did you expect to happen**:

  -A test notification email is received by the email contact point.

### 
# How do we reproduce it?

**Step 1**:
Change smtp configuration in grafana.ini with the following for a non-tls smtp server:
`[smtp]
enabled = true
host = internal.server.net:25
user = smtpuser
# If the password contains # or ; you have to wrap it with triple quotes. Ex """#password;"""
password = smtppassword
;cert_file =
;key_file =
skip_verify = false
from_address = grafana@internal.com
from_name = Grafana
# EHLO identity in SMTP dialog (defaults to instance_name)
;ehlo_identity = dashboard.example.com
# SMTP startTLS policy (defaults to 'OpportunisticStartTLS')
;startTLS_policy = NoStartTLS`

**Step 2**:

Make a notification test to an email

**Step 3**:

The error message appears:
![image](https://github.com/grafana/grafana/assets/65592618/cea69e0c-8b92-45d1-a4cc-21a5a1a5174c)
Grafana log shows the message:
```
logger=ngalert.notifier.email t=2023-06-06T15:08:14.761627379Z level=error msg="Missing receiver"
logger=ngalert.notifier.email t=2023-06-06T15:08:14.761785522Z level=error msg="Missing group labels"
```

### 
# What Grafana version are you using?

v9.5.0

### 
### Is the bug inside a Dashboard Panel?

Copy the panel's ["get-help" data](https://grafana.com/docs/grafana/latest/troubleshooting/send-panel-to-grafana-support/) here

### Grafana Platform?

A package manager (APT, YUM, BREW, etc.)

### User's OS?

Ubuntu 22.04

### Is this a Regression?

None

**Comment by grobinson-grafana** [comment]
Hi! :wave: Grafana does not allow you to send your SMTP username/password over an unencrypted connection. I would recommend using TLS.

**Comment by grobinson-grafana** [comment]
> Hi @grobinson-grafana I'm having similar issues with this, however using TLS with an AWS SES server set-up. I am getting a 'Test alert sent' from the UI, but nothing sent to my email (previously when misconfigured SES creds it failed with unauthorized errors so believe all is now okay SMTP/aws side).
> 
> Grafana container logs are only showing: logger=alerting.notifier.email t=2023-06-07T13:37:42.741026847Z level=error msg="Missing receiver" logger=alerting.notifier.email t=2023-06-07T13:37:42.741070761Z level=error msg="Missing group labels"
> 
> and nothing else. I'm on v9.2.2 and have seen many others having similar issues. Is there a version known to have sorted this issue out? thank you in advance!

Hi! Can you please open a separate issue for this as it seems unrelated to plain text authentication in SMTP.

**Comment by armandgrillet** [comment]
Closing as no answer from original author, please reopen if the issue is still there after following our help.

**Comment by forrejam** [comment]
I would ask this issue be reopened. Using unencrypted and unauthenticated SMTP is extremely common in enterprise environments, not having it as an option is a detriment to Grafana and requires hacky work arounds such as local SMTP relays.
